cordova-plugin-mobilespec-tests/tests/whitelist.tests.js

```diff
@@ -202,7 +202,7 @@ exports.defineAutoTests = function () {
             itShouldReject('http://www.apache.org.evil.com/');
             itShouldAccept('file:///foo');
             if (cordova.platformId == 'android')
-                itShouldAccept('content:///foo');
+                itShouldReject('content:///foo');
```

Comment:
It has been changed in https://github.com/apache/cordova-mobile-spec/commit/b8401eb3c99573de6e6ae6af952f32d19bd2cd28#diff-e3ccd521f7df2ab57e6ee9133e737b36R205
I have asked the author of this change if this is intentional but received no reply.
I suppose that it was not intentional so I am reverting that change here.